CWEFanFork/common/countries/Berlin.txt

```diff
@@ -55,6 +55,17 @@ party = {
 		start_date = 1836.1.1
 		end_date = 2092.1.1
 }
+party = {
+		name = "social_democrat_party"
+		citizenship_policy = full_citizenship
+		economic_policy = state_capitalism
+		ideology = social_democrat
+		religious_policy = secularized
+		trade_policy = protectionism
+		war_policy = pacifism
+		start_date = 1836.1.1
+		end_date = 2092.1.1
+}
 party = {
 		name = "SDP - Berlin"
 		citizenship_policy = limited_citizenship
```

Comment:
I believe I meant to make this party social_democrat, but must've forgotten to change the ideology.

Reply:
fixed